README.md

```diff
@@ -75,7 +75,7 @@ forwardable = "SUBSCRIBE" SP id [ SP "PRESENCE" ]
 
 compat      = verb [ SP id ] [ SP payload ]
 
-code        = 3DIGIT
+code        = 3*DIGIT
```

Comment:
A response code is *exactly* 3 DIGIT characters. The proposed change would make it  3 or more characters, which would be incorrect.